### ISSUE TITLE
[469b/469c/469d] Flag capture sounds too quiet

### ISSUE THREAD

**Issue by user14159** [OldUnreal/UnrealTournamentPatches#505]
The flag capture sounds in 469b with Galaxy, Cluster, and OpenAL sound renderers are much too quiet. The following is a video using Galaxy audio for both patches with identical volume settings:

https://www.youtube.com/watch?v=5Rudy_tPjKA

436
0:11 Sound settings
0:40 Blue capture
0:54 Red capture

469b
1:13 Sound settings
2:11 Red capture
2:30 Blue capture

**Comment by SeriousBuggie** [comment]
Not able reproduce it on v469b or v469c clients with v469c server.

**Comment by user14159** [comment]
Exact same problem with 469c.

**Comment by stijn-volckaert** [comment]
We landed a last-minute fix for this problem in the 469d release build

**Comment by user14159** [comment]
I know you closed this but it still doesn't seem fixed to me. Sounds are somewhat subjective and I wanted to give a bit more rigorous demonstration of the problem. I made two audio recordings, one on v436 and one on v469d. Both were using Galaxy audio with the exact same settings. Windows volume was set to 20 for both. The recording consisted of a single enforcer shot as a control for volume levels, some footsteps, and a capture of the blue flag. I then imported these audio files into Audacity side by side and took a screenshot of the result. As you can see, all the sounds seem to be at a similar level _**except for the flag capture sound**_. Maybe this issue could be reopened?

![469](https://github.com/OldUnreal/UnrealTournamentPatches/assets/17584191/dfb1fafe-e1ea-45cc-99c5-dde85411f8b5)

**Comment by SeriousBuggie** [comment]
It is local play or online play?

**Comment by user14159** [comment]
Practice mode

**Comment by SeriousBuggie** [comment]
If play on dedicated server it fine or too as in practice?

**Comment by user14159** [comment]
It's a problem everywhere. I did my tests offline to isolate the issue and show that it has nothing to do with the server.

**Comment by SeriousBuggie** [comment]
There exists difference from which Actor played sounds on online play and locally. That why I ask you check locally.
Look like our fix not work for local case. Condition not meet. Possible online, it be fine. That why it need check, for clarity.

**Comment by user14159** [comment]
It most definitely has not been fixed, online or offline.

**Comment by user14159** [comment]
It was not possible for me to run the test on an independent server as the ones I tried either have the IG+ mod, which is not backwards compatible with v436, or use an outdated version of ACE which kicks for v469d. 

However, I set up my own dedicated server and connected to it through the LAN browser on the same machine. I'm not sure if this meets your qualifications for a dedicated server or not, but the results were the exact same.

![469d](https://github.com/OldUnreal/UnrealTournamentPatches/assets/17584191/edc42ade-14a5-415c-8176-c224379798d3)
